bazel/emscripten_toolchain/emscripten.BUILD

```diff
@@ -1,6 +1,41 @@
 package(default_visibility = ['//visibility:public'])
 
 filegroup(
-    name = "all",
-    srcs = glob(["**"]),
+    name = "includes",
+    srcs = glob([
+        "emscripten/cache/sysroot/include/c++/v1/**",
+        "emscripten/cache/sysroot/include/compat/**",
+        "emscripten/cache/sysroot/include/**",
+        "lib/clang/15.0.0/include/**",
+    ]),
+)
+
+filegroup(
+    name = "compiler_files",
+    srcs = [
+        "emscripten/emcc.py",
+        "bin/clang",
+        "bin/clang++",
+        ":includes",
+    ],
+)
+
+filegroup(
+    name = "linker_files",
+    srcs = [
+        "emscripten/emcc.py",
+        "bin/llvm-nm",
+        "bin/llvm-objcopy",
+        "bin/wasm-emscripten-finalize",
+        "bin/wasm-ld",
+        "bin/wasm-opt",
```

Comment:
What about the libraries (from the sysroot) that are used during linking?  Are they handled separately?